make/test.mk

```diff
@@ -35,7 +35,18 @@ test-e2e: deploy-e2e e2e-run
 
 .PHONY: deploy-e2e
 deploy-e2e: INSTALL_OPERATOR=true
-deploy-e2e: build clean-e2e-files deploy-host deploy-members e2e-service-account setup-toolchainclusters
+deploy-e2e: build clean-e2e-files label-olm-ns deploy-host deploy-members e2e-service-account setup-toolchainclusters
+
+label-olm-ns: create-host-project create-member1 create-member2
+# adds a label on the oc label ns/openshift-operator-lifecycle-manager name=openshift-operator-lifecycle-manager
+# so that deployment also works when network policies were configured with `sandbox-cli`
+	@-oc label --overwrite=true ns/openshift-operator-lifecycle-manager name=openshift-operator-lifecycle-manager
+	@echo "adding network policies in $(HOST_NS) namespace"
+	@-oc process -p NAMESPACE=$(HOST_NS) -f ${PWD}/make/resources/default-network-policies.yaml | oc apply -f -
+	@echo "adding network policies in $(MEMBER_NS) namespace"
+	@-oc process -p NAMESPACE=$(MEMBER_NS) -f ${PWD}/make/resources/default-network-policies.yaml | oc apply -f -
+	@echo "adding network policies in $(MEMBER_NS_2) namespace"
+	@-oc process -p NAMESPACE=$(MEMBER_NS_2) -f ${PWD}/make/resources/default-network-policies.yaml | oc apply -f -
```

Comment:
There is one thing I just realized - in dev environment, we don't create the second member namespace and we even don't set the `MEMBER_NS_2` var. So I'm afraid that it would fail in this case

Reply:
btw, we already have `create-project` target - why not moving the creation of the NetworkPolicies there?

Reply:
good point. I wanted to be sure that the network policies would exist before the deployment (otherwise this latter may fail), but setting the policies in the `create-project` target looks way simpler and straightforward, indeed! thanks!

Reply:
> There is one thing I just realized - in dev environment, we don't create the second member namespace and we even don't set the `MEMBER_NS_2` var. So I'm afraid that it would fail in this case

yes, let me handle this case, too

Reply:
done in 70bceb2